rl_loop/update_resign_threshold.py

```diff
@@ -0,0 +1,83 @@
+# Copyright 2018 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Updates a flagfile containing a resign threshold flag
+
+Reads the bigtables defined by env vars in PROJECT,CBT_INSTANCE,CBT_TABLE to
+compute the 95 percentile of the bleakest-evaluations found in calibration games,
+then updates the flagfile on the default bucket path, resetting that value
+"""
+
+import sys
+import re
+import os
+import time
+from absl import app
+from absl import flags
+import numpy as np
+import tensorflow as tf
+
+sys.path.insert(0, '.')
```

Comment:
I assume this is needed because `bigtable_input` isn't located sibling to this module, but that the script gets run from the top-level minigo directory. Is this a pattern used elsewhere here?

Reply:
that answer isn't actually right?